<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addToMasterDiag</class>
 <widget class="QDialog" name="addToMasterDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>PID:            </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPID"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Data:          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditData"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Conversion:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConversion"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Comments: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditComments"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Year:           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditYear"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Make:         </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMake"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Model:        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditModel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>VIN:            </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditVIN"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOkay">
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
